src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregnYtelseSteg.kt

```diff
@@ -63,6 +65,8 @@ class TilsynBarnBeregnYtelseSteg(
     }
 
     private fun beregnOgLagreOpphør(saksbehandling: Saksbehandling, vedtak: OpphørTilsynBarnDto) {
+        opphørValidatorService.validerOpphør(saksbehandling)
```

Comment:
Nå beregnes det to ganger siden det også skjer i valideringen. Kunne kanskje splittet det opp, så man først validerer ingen nye og ingen oppfylte etter revurder fra, beregner, og til slutt validerer at det ikke er noen beløpsperioder etter opphørsdato

Reply:
Takk!

Reply:
b6b2dd88f5017ed17ab08ee076b44833539e0922